exotica/doc/index.rst

```diff
@@ -32,28 +33,51 @@ Overview
 -  `Setting up ROSlaunch <Setting-up-ROSlaunch.html>`__
 
 
+Setup
+=====
+
 .. toctree::
    :maxdepth: 2
-   :caption: Contents:
 
    Installation
+
+Basics
+======
+
+In this section we'll explore the basics of setting up and using 
+EXOTica with the included KUKA LWR_Simplified robot.
+
+.. toctree::
+   :maxdepth: 2
+
+   Quickstart-python
+   Quickstart-cpp
+   Setting-up-problems-and-solvers
+   Using-EXOTica
+
+
+Advanced
+========
+
+With the advanced section, we look at different methods of initializing
+EXOTica and using it with your own robot. 
+
+.. toctree::
+   :maxdepth: 2
+
    Prerequisites
    Initialization
-   Initializers
    XML
    XML-Parsing
    Manual-Initialization
    Common-Initialization-Step
-   Quickstart
-   Quickstart-cpp
-   Setting-up-problems-and-solvers
-   Task_maps
    Setting-up-ROSlaunch
-   Using-EXOTica
-   Python-API
-   Documentation
+   Task_maps
+   Initializers
```

Comment:
Whis has this been moved so low if manual initialization is above? Perhaps group task maps and initializers under core components / concepts